src/styles/blocks/card.css

```diff
@@ -0,0 +1,111 @@
+body {
+  background-color: var(--midium-bg-color);
+}
+
+.card {
+  min-width: 200px;
+  display: flex;
+  padding: calc(2 * var(--card-standard-padding))
+    var(--card-standard-padding)
+    var(--card-standard-padding);
+  flex-direction: column;
+  margin-left: 56px;
+  width: 200px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+
+  background-color: var(--light-bg-color);
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --light-bg-color: #fff;
```

Comment:
Don't declare variables in one of the styling rules, the purpose of variables is to have them easily accessible to change how your current design behaves, it would be best to create a separate file variables.css and declare them there in :root